### ISSUE TITLE
Add App Notification Control

### ISSUE THREAD

**Issue by southern-dust** [no-go/NotifyRelay#11]
As tittle.
Add Notification Control For Each App to decide which App Notification could be forward.
Though Android has given us Notification Control settings on Android Settings, I think it still necessary because it's not conflict between what we want to see and what we want to forward.
Also, it could enhanced as an Advanced Feature of this App.

**Comment by no-go** [comment]
good point. that is very similar to my blacklist/whitelist idea. maybe a list of application with checkboxisis a better/nice feature.

**Comment by southern-dust** [comment]
Yeah!
As an advanced feature, it's not only decide which app, but also which Notifications in an App would be forward.

That also a feature above Android 7.x.

It looks like so familar with Android System's feature, isn't it?

What's the better way?

**Comment by no-go** [comment]
Did you mean: Notification priorities or groups or Notification Channels?

There are some problems with it:

- I have a big focus on compatibility to old phones and android versions
- The complexity for the GUI to configure this stuff gets high
- The portfolio of example code with a focus on notification channels is low

Additionally: I hate the notification system and concept > Android 6 and had bad experience with it. Maybe a Black/Whitelist on Buzzwords and a list with App names breaks it all down to a good workaround.

I am sorry, but most of my apps are all-day-hacks for my usage and are similar to a proof of concept. Thus: I did not spend my primary time to make a single app "perfect". This is certainly frustrating for you and others, but I always hope that the OpenSource community picks up apps from me and their ideas and develops them further. So far I don't experience such a tendency.

If you find an opensource app with the possibility to select special notificaions of a special (alien) App, please show me that App. The "Android System Feature" you told about is quite not the same, because System features have more rights and access to the notification engine.

:o/

**Comment by southern-dust** [comment]
Well. I'm sorry for tending you add more complexity in this App.

Maybe you are right, a simple Black/Whitelist with App names would make it simple but powerful.

If we wanna more advanced control of per app Notifications, system will provide more powerful ability.

Thanks for developing such an useful app.
